operator-examples/myrealm.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: keycloak.org/v1alpha1
+kind: KeycloakRealm
+metadata:
+  name: myrealm
+  labels:
+    realm: myrealm
+spec:
+  realm:
+    id: "myrealm"
+    realm: "myrealm"
+    enabled: True
```

Comment:
A shame this is needed, as it really should be enabled by default ;)

Reply:
+1